soc/HPM6280/CMakeLists.txt

```diff
@@ -32,6 +32,22 @@ endif()
 
 sdk_nds_compile_options(-mcpu=d45)
 
+if("${TOOLCHAIN_VARIANT}" STREQUAL "zcc")
+    sdk_compile_options(--target=riscv32-unknown-elf)
+    sdk_compile_options(-march=rv32imafdcxandes)
+    sdk_compile_options(-mabi=ilp32d)
```

Comment:
FPU will cause significant isr latency due to context saving, is it any specific reason to enable FPU by default?